boards/st/nucleo_n657x0_q/doc/index.rst

```diff
@@ -0,0 +1,169 @@
+  of the data brief for details);
+
+  - External or internal SMPS to generate Vcore logic supply
+  - Ethernet compliant with IEEE-802.3-2002
+  - USB Device only, USB OTG full speed, or SNK/UFP (full-speed or high-speed mode)
+  - Board connectors:
+
+    - ARDUINO® Uno V3 connector or ST Zio expansion connector including ARDUINO® Uno V3
+    - Camera module FPC
+    - MIPI20 compatible connector with trace signals
+    - USB with Micro-AB or USB Type-C®
+    - Ethernet RJ45
+
+  - On-board ST-LINK (STLINK/V2-1, STLINK-V3E, or STLINK-V3EC) debugger/programmer with
+    USB re-enumeration capability: mass storage, Virtual COM port, and debug port
+
+Supported Features
+==================
+
+The Zephyr ``nucleo_n657x0_q`` board supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+:zephyr_file:`boards/st/nucleo_n657x0_q/nucleo_n657x0_q_defconfig`
+
+
+Connections and IOs
+===================
+
+NUCLEO_N657X0_Q Board has 12 GPIO controllers. These controllers are responsible
+for pin muxing, input/output, pull-up, etc.
+
+For more details please refer to `NUCLEO_N657X0_Q User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- LD1 : PO1
+- LD2 : PG10
+- USART_1_TX : PE5
+- USART_1_RX : PE6
+
+System Clock
+------------
+
+NUCLEO_N657X0_Q System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by PLL clock at
+400MHz, driven by 64MHz high speed internal oscillator.
+
+Serial Port
+-----------
+
+NUCLEO_N657X0_Q board has 10 U(S)ARTs. The Zephyr console output is assigned to
+USART1. Default settings are 115200 8N1.
+
+Programming and Debugging
+*************************
+
+NUCLEO_N657X0_Q board includes an ST-LINK/V3 embedded debug tool interface.
+This probe allows to flash the board using various tools.
+
+Flashing
+========
+
+The board is configured to be flashed using west `STM32CubeProgrammer`_ runner,
+so its :ref:`installation <stm32cubeprog-flash-host-tools>` is required.
```

Comment:
Please mention that version v2.18.0 or later is required

Reply:
Done